src/app/pages/communication/components/privacy-notice/privacy-notice.ts

```diff
@@ -14,12 +15,30 @@ export class PrivacyNoticeComponent implements OnInit {
   language: Language;
 
   @Input()
-  isRider: boolean = false;
+  category: CategoryCode;
 
   constructor(private translate: TranslateService) { }
 
   ngOnInit() : void {
     configureI18N(this.language, this.translate);
   }
 
+  categorySpecificTestContext(cat: CategoryCode): string {
+    switch (cat) {
+      case 'ADI3':
+        return 'ADI';
+      case 'EUA1M1':
+      case 'EUA1M2':
+      case 'EUA2M1':
+      case 'EUA2M2':
+      case 'EUAM1':
+      case 'EUAM2':
+      case 'EUAMM1':
+      case 'EUAMM2':
+        return 'common.riding';
```

Comment:
can we use the ```TestCategory``` enum in this switch please

Reply:
good shout